<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF小工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_merge">
    <property name="title">
     <string>合并PDF</string>
    </property>
    <addaction name="action_merge"/>
   </widget>
   <widget class="QMenu" name="menu_convert">
    <property name="title">
     <string>PDF图片工具</string>
    </property>
    <addaction name="action_extract"/>
    <addaction name="action_convert"/>
   </widget>
   <addaction name="menu_merge"/>
   <addaction name="menu_convert"/>
   <addaction name="menu_help"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_merge">
   <property name="text">
    <string>合并PDF</string>
   </property>
  </action>
  <action name="action_extract">
   <property name="text">
    <string>提取图片</string>
   </property>
  </action>
  <action name="action_convert">
   <property name="text">
    <string>转图片</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
